### DISCUSSION TITLE
Github Wiki - .md footnote not supported

### DISCUSSION THREAD

**Discussion by ken-mellem** [community/community#56433] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I find it odd that footnotes [^footnote] is not supported in the wiki.

[^footnote]: it has several usages.

Any particular reason why this is not a feature supported in the Github Wiki? I mean, it's markdown after all... https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/  or?

**Comment by jc00ke** [comment]
I just ran into this and would love to see footnotes supported in the wiki as well.

**Comment by michaelahlers** [comment]
I need this, too—it'd be incredibly helpful for a guide project I'm working on.

**Comment by trrapp12** [comment]
I have also failed to get it to work and would love to see it added.

**Comment by JetForMe** [comment]
Why on Earth is it supported everywhere else, but not in the Wiki?

**Comment by yash509** [comment]
Same issue with me too

**Comment by cjcoats** [comment]
I find it buggy:  the MD source
[^1]: The Fortran Standard explicitly **refuses** to dictate the quality
of how round-off behaves.  As an extreme example, *no two of the
following* are guaranteed to be equal (a problem found especially on
Cray vector, IBM mainfreame and POWER, Intel x86/x87, and SGI platforms),
although the naive impression is that they should all be equal:
<pre>
        INTEGER, PARAMETER :: A = 1.0 / 3.0
        INTEGER, PARAMETER :: B = 0.333333333333333333  ! with extra-digit overkill
        INTEGER, PARAMETER :: C = FLOAT( 1 ) / FLOAT( 3 )
        INTEGER     D, E, F, G
        ...
        D = 1.0 / 3.0
        E = FLOAT( 1 ) / FLOAT( 3 )
        F = 0.333333333333333333
        READ( '0.333333333333333333', * )  G
        ...
</pre>

displays as though the <pre>...<pre> was placed before the Note 1.  See attached.
![NOTES](https://github.com/community/community/assets/10758493/e30ff359-780d-480c-b6df-7b9b04edbc04)
![NOTES-MD](https://github.com/community/community/assets/10758493/f33f9f05-0ba9-433e-bd07-9579029b1bd8)

**Reply by dreua** [reply]
I think this is not about the wiki, so different issue in the same broader context.

**Comment by dreua** [comment]
This is frustrating, bad UX. The blog post of course doesn't mention it either.

At least the documentation is correct, but the majority of people will read that only after they tried it and failed.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes

**Comment by mkurz** [comment]
Just bumped into that as well. Please support footnotes in wikis, thanks!

**Comment by taprs** [comment]
Bump! This is needed.

**Comment by williamrowell** [comment]
Yes, Please support footnotes in wiki markdown.

**Comment by JochemPalmsens** [comment]
Bump!

**Reply by dreua** [reply]
I like to subscribe to issues like this in case anything meaningful or actual updates are communicated here. While I understand your intention to "bump" this, it is annoying noise for people like me and it is highly unlikely that anyone with the the power to fix this will see or react to your bumping. The upvote, reactions and "subscribe" button is the right thing to do if you want to support this issue without unnecessary noise. Maybe try other ways to contact Github employees or gain publicity for this issue, the people subscribed here are already aware of it :wink:

**Comment by johncarney** [comment]
+1

**Reply by dreua** [reply]
I like to subscribe to issues like this in case anything meaningful or actual updates are communicated here. While I understand your intention to "bump" this, it is annoying noise for people like me and it is highly unlikely that anyone with the the power to fix this will see or react to your bumping. The upvote, reactions and "subscribe" button is the right thing to do if you want to support this issue without unnecessary noise. Maybe try other ways to contact Github employees or gain publicity for this issue, the people subscribed here are already aware of it 😉

**Comment by melhajj06** [comment]
bump

**Reply by dreua** [reply]
I like to subscribe to issues like this in case anything meaningful or actual updates are communicated here. While I understand your intention to "bump" this, it is annoying noise for people like me and it is highly unlikely that anyone with the the power to fix this will see or react to your bumping. The upvote, reactions and "subscribe" button is the right thing to do if you want to support this issue without unnecessary noise. Maybe try other ways to contact Github employees or gain publicity for this issue, the people subscribed here are already aware of it 😉

**Comment by MikeWeller** [comment]
I would also like this. I was surprised to find it worked in other areas e.g. gists but not wikis.
